pkg/plugins/resources/k8s/store.go

```diff
@@ -75,11 +79,27 @@ func (s *KubernetesStore) Create(ctx context.Context, r core_model.Resource, fs
 	return nil
 }
 
+func markAsSynced(obj k8s_model.KubernetesObject) {
+	annotations := obj.GetAnnotations()
```

Comment:
Maybe Labels?

Reply:
This functionality seems very internal. I think it's more of an annotation